docs/shortcuts.rst

```diff
@@ -0,0 +1,40 @@
+.. _layers:
+
+Keyboard shortcuts
+==================
+
+In both widgets, viewer navigation with the mouse follows typical conventions --
+click and drag up or down to move vertically; click and drag left or right to
+move horizontally; mouse wheel or pinch/two-finger scroll on a touchpad to zoom.
+
+In the Qt widget, it is also possible to navigate the viewer via keyboard
+shortcuts that are further explained below.  
+
+Basic movements
+---------------
+
+Imagining that the viewer's movements correspond with `an aircraft's principal
+axes of rotation <https://en.wikipedia.org/wiki/Aircraft_principal_axes>`_, you
+can perform two of the basic movements with the arrow keys alone:
+
+``up``, ``down``: pitch
+``left``, ``right``: yaw
+
+To zoom, use:
+
+``z``: zoom in
+``x``: zoom out
+
+Modified movements
+------------------
+
+``alt`` allows you to access the third axis of rotation:
+``alt`` + ``left``, ``alt`` + ``right``: roll
+
+In individual planet modes, ``alt`` + ``up`` or ``down`` tilts the body
+vertically.
+
+Finally, adding ``shift`` to any of the above keys or combinations acts as a
+fine adjustment switch, performing the original action more slowly. For example,
+``shift`` + ``z`` zooms in at a slower pace, and ``shift`` + ``alt`` + ``left``
+gives you a slower roll motion.
```

Comment:
I think it might be easier to simply list the shortcuts in a table - see:

http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables

For how to do this in rst. Then maybe under the table you can keep the paragraph about the shift key to avoid repeating everything.